src/server/models/flight.js

```diff
@@ -26,4 +31,12 @@ FlightSchema.statics = {
   }
 };
 
-export default mongoose.model('Flight', FlightSchema);
+FlightSchema.statics.getFlightFromFlightName = (flightname) => {
+  const flightNameArray = flightname.split('-');
```

Comment:
Yes, some guard against garbage input would be good here, since we are essentially serializing two data together with a hyphen delimiter.